internal/storetesting/charm-repo/quantal/juju-gui/README.md

```diff
@@ -0,0 +1,200 @@
+<!--
```

Comment:
no realistically its just a directory full of charms, If you look quantal isn't in the supported series.